internal/handler/ticket.go

```diff
@@ -0,0 +1,37 @@
+package handler
+
+import (
+	"context"
+	"log/slog"
+	"net/http"
+
+	"github.com/dilyara4949/flight-booking-api/internal/domain"
+	"github.com/dilyara4949/flight-booking-api/internal/handler/response"
+	"github.com/dilyara4949/flight-booking-api/internal/middleware"
+	"github.com/gin-gonic/gin"
+	"github.com/google/uuid"
+)
+
+type TicketService interface {
+	GetAll(ctx context.Context, userID uuid.UUID, page, pageSize int) ([]domain.Ticket, error)
```

Comment:
I would suggest either renaming it to "GetTickets", "GetList" or similar, since we applying pagination here